recipe/meta.yaml

```diff
@@ -5,15 +5,17 @@ package:
   version: {{ version }}
 
 source:
-  url: https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-{{ version }}-linux-x86_64.tar.gz  # [linux]
-  sha256: b24ba27d57463e08db859b7941cf4d7c33cb3af5865095639e3e0b2055cbec66  # [linux]
-  url: https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-{{ version }}-darwin-x86_64.tar.gz  # [osx and build_platform == "osx-64"]
-  sha256: 85d71c956529453f246e3b41339903b8992aa326f15ea6bc36f2ec79e76c7d02  # [osx and build_platform == "osx-64"]
-  url: https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-{{ version }}-darwin-arm.tar.gz  # [osx and build_platform == "osx-arm64"]
```

Comment:
Shouldn't this be dependent on the target platform and not the build? given that the `osx_arm64` target is built `osx_64`.